perch/organisation.py

```diff
@@ -55,8 +55,8 @@ class Organisation(Document):
     internal_fields = Document.internal_fields + ['services', 'repositories']
     read_only_fields = ['created_by']
 
-    @classmethod
-    def schema(cls, doc):
+    @property
+    def schema(cls):
```

Comment:
`property` only works on instances, so the convention is to use `self` instead of `cls`